app/components/works/detail_component.html.erb

```diff
@@ -0,0 +1,214 @@
+<section class="details">
```

Comment:
Moved this in from `app/views/works/show.html.erb` so that we can easily test the new condition.

Reply:
:ok_hand: